gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposeCompilerCompatibility.kt

```diff
@@ -16,7 +16,8 @@ internal object ComposeCompilerCompatibility {
         "1.9.0-RC" to "1.4.8-beta",
         "1.9.0" to "1.5.1",
         "1.9.10" to "1.5.2",
-        "1.9.20-Beta" to "1.5.2.1-Beta2"
+        "1.9.20-Beta" to "1.5.2.1-Beta2",
+        "1.9.20-Beta2" to "1.5.2.1-Beta3"
     )
 
     fun compilerVersionFor(kotlinVersion: String): String {
```

Comment:
We have a failed test. Does it fail before the change?

Reply:
I'll check.
This change doesn't affect the version of the plugin used in the tests. So it's likely a flaky test/environemnt.

Reply:
rerun helped :|